frontend/public/models/yaml-templates.ts

```diff
@@ -1193,18 +1193,16 @@ spec:
 `,
   );
 
-const pluginTemplates = ImmutableMap<
-  GroupVersionKind,
-  ImmutableMap<string, string>
->().withMutations((map) => {
-  plugins.registry.getYAMLTemplates().forEach((yt) => {
-    const modelRef = referenceForModel(yt.properties.model);
-    const templateName = yt.properties.templateName || 'default';
+export const getYamlTemplates = (extensionTemplates: YAMLTemplate[]) =>
```

Comment:
For those callers that don't need to work with plugin contributed templates:

```suggestion
export const getYAMLTemplates = (extensionTemplates: YAMLTemplate[] = []) =>
```